### PR TITLE
Make everything portable

### PR DESCRIPTION
**Pull request by BrennanConroy**
@pakrym @JunTaoLuo @Tratcher

### CODE REVIEW

File: samples/Builder.Filtering.Web/web.config [hunk 1]
```diff
@@ -1,9 +0,0 @@
-﻿<?xml version="1.0"?>
```
Comment (Tratcher) [inline samples/Builder.Filtering.Web/web.config]: Pending dicsussion on web.config here: https://github.com/aspnet/IISIntegration/pull/143#discussion_r60499936

File: samples/Logging.Elm/project.json [hunk 2]
```diff
@@ -4,16 +4,19 @@
     "emitEntryPoint": true
   },
   "dependencies": {
+    "Microsoft.NETCore.Platforms": "1.0.1-*",
```
Comment (Tratcher) [inline samples/Logging.Elm/project.json]: nit: Why is this sorted to the top rather than the bottom?

Comment (BrennanConroy) [inline samples/Logging.Elm/project.json]: @pakrym ?

Comment (pakrym) [inline samples/Logging.Elm/project.json]: No big reason, automation did it this way.

**Comment by BrennanConroy** [comment]
This should have been opened on release

**Comment by muratg** [comment]
This PR should be sent to release branch (and merged to dev later)

**Comment by BrennanConroy** [comment]
:up: :date:

File: samples/Data.InMemory/project.json [hunk 3]
```diff
@@ -3,14 +3,17 @@
     "emitEntryPoint": true
   },
   "dependencies": {
-    "Microsoft.EntityFrameworkCore.InMemory": "1.0.0-*",
-    "Microsoft.NETCore.Platforms": "1.0.1-*"
```
Comment (Tratcher) [inline samples/Data.InMemory/project.json]: Revert these ordering changes, they're wrong and unnecessary.

**Comment by Tratcher** [comment]
:shipit:

File: samples/Config.ChangeNotification.Web/project.json [hunk 4]
```diff
@@ -1,15 +1,12 @@
 {
   "dependencies": {
     "Microsoft.AspNetCore.Http": "1.0.0-*",
-    "Microsoft.Extensions.Configuration": "1.0.0-*",
-    "Microsoft.NETCore.Platforms": "1.0.1-*"
+    "Microsoft.Extensions.Configuration": "1.0.0-*"
```
Comment (JunTaoLuo) [inline samples/Config.ChangeNotification.Web/project.json]: Why did you remove `"Microsoft.NETCore.Platforms": "1.0.1-*"`? It's needed for `net451`

Comment (BrennanConroy) [inline samples/Config.ChangeNotification.Web/project.json]: A script did it, idk how :(

Comment (JunTaoLuo) [inline samples/Config.ChangeNotification.Web/project.json]: Whose script?

Comment (BrennanConroy) [inline samples/Config.ChangeNotification.Web/project.json]: Might be because Microsoft.AspNetCore.Http or Microsoft.Extensions.Configuration brought it in

Comment (BrennanConroy) [inline samples/Config.ChangeNotification.Web/project.json]: Oh this one doesn't have anything to run

Comment (JunTaoLuo) [inline samples/Config.ChangeNotification.Web/project.json]: Oh right...
